integration-tests/chopsticks/src/network/polkadot.ts

```diff
@@ -27,11 +27,7 @@ export function removeDisputesAndMessageQueues() {
 		ParasDisputes: {
 			// those can makes block building super slow
 			$removePrefix: ['disputes'],
-		},
-		Dmp: {
```

Comment:
@Ad96el maybe this needs to be replaced by some other storage entry in the XCMP pallet?

Reply:
Not sure. The pallet is still available on the relay chain, and I also checked the implementation in chopsticks. They still have this replacement in their testing [repository](https://github.com/AcalaNetwork/e2e-tests/blob/master/networks/polkadot.ts). I would probably leave it like that.

Reply:
Yes but this pallet won't be in the runtime anymore. It does not make sense to have it in here. Either an alternative storage is used, or this makes no sense to me. I think they probably just forgot.

Reply:
Sorry didn't see this is relaychain stuff. I added them back: https://github.com/KILTprotocol/kilt-node/pull/723/commits/2d5bfb3b76912c61641286866564a8add21ca779